migration/sql-files/006-external-provider.sql

```diff
@@ -0,0 +1,22 @@
+
+-- token storage
+
+CREATE TABLE external_providers (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    type text,
+    url text
+);
+
+CREATE TABLE external_provider_tokens (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
```

Comment:
Do we actually want to support soft deletes here? Maybe it's safer to allow to remove tokens completely? Not quite comfortable to keep tokens in the DB forever.

Reply:
We could definitely hard-delete tokens - but was wondering if hard-delete in general should be a 'cleanup' job for the entire system.